TestFramework/TestSupportFiles/ToDo.cs

```diff
@@ -17,6 +17,9 @@ public class ToDo : Entity
 
 		[JsonProperty("value")]
 		public int Value { get; set; }
+
+		[JsonProperty("bool_value")]
+		public bool BoolVal { get; set; }
```

Comment:
@vinaygahlawat Just as test completeness, will it matter if this datatype is either `bool` or `Boolean`? Do we have a test that covers `Boolean`?